jupyterhub_moss/form/option_form.js

```diff
@@ -7,27 +7,33 @@ function removeAllChildren(node) {
   }
 }
 
-function createEnvironmentDiv(key, description, path, checked = false) {
+function createEnvironmentDiv(key, description, path, modules, checked = false) {
   const div = document.createElement('div');
   div.classList.add('environment-div');
 
-  // Store env key in hidden input
-  const hidden_input = document.createElement('input');
-  hidden_input.setAttribute('type', 'hidden');
-  hidden_input.setAttribute('value', key);
-  div.appendChild(hidden_input);
-
   const radio_id = `environment_radio_${key}`;
-  const title = `Environment path: ${path}`;
+  const title = `Environment Name: ${description}`;
```

Comment:
`description` is provided in the label, let's put additional info in the tooltip:
```suggestion
  const title = [path ? `Path: ${path}` : "", modules ? `Modules: ${modules}` : ""].filter(v => v).join("\n");
```